### DISCUSSION TITLE
Proposition to make a discord server.

### DISCUSSION THREAD

**Discussion by zeykafx** [bttv-android/bttv#146] [Ideas]
Hey!

I'm really interested in this project and I really want to help (I'd like to contribute some code, I've been modifying a few apps here and there, not sure if I can help) and I think it would be nice to have a discord server where people can chat about the mod, ask for new features, report bugs or weird behavior,...

I'm ready to help with the set up and stuff if you are not comfortable with discord.

What do you think?

**Comment by zeykafx** [comment]
@FoseFx What do you think? I can set it up if you don't want to do it.

**Reply by FoseFx** [reply]
I don't think the project is big enough but why not: https://discord.gg/7jgDGdXXqN
